### PR TITLE
Ann

### PR DESCRIPTION
**Pull request by annperishko**

### CODE REVIEW

File: src/main/java/model/AbstractEntity.java [hunk 1]
```diff
@@ -1,4 +1,13 @@
 package model;
 
 public class AbstractEntity {
+    private int id;
```
Comment (KrasuliaV) [inline src/main/java/model/AbstractEntity.java]: For id use Long better

File: src/main/java/model/Order.java [hunk 2]
```diff
@@ -1,4 +1,45 @@
 package model;
 
+import java.util.ArrayList;
+
+
 public class Order extends AbstractEntity {
+
+    private String ownerName;
+    private boolean status;
+    private ArrayList<String> product;
```
Comment (KrasuliaV) [inline src/main/java/model/Order.java]: What do you want to hold in this ArrayList - Product or productName?

File: src/main/java/model/User.java [hunk 3]
```diff
@@ -1,41 +1,66 @@
...
 
+import java.util.ArrayList;
+
 public class User extends AbstractEntity {
     private String userName;
     private String password;
-    private boolean status;
+    private boolean isBlocked;
+    private UserRole role;
+    private ArrayList<String> order;
```
Comment (KrasuliaV) [inline src/main/java/model/User.java]: In this ArrayList, you must hold Orders, not String

File: src/main/java/model/User.java [hunk 4]
```diff
@@ -1,41 +1,66 @@
...
 
+    public UserRole getRole() {
+        return role;
+    }
+
+    public void setRole(UserRole role) {
+        this.role = role;
+    }
+
+    enum UserRole {
```
Comment (KrasuliaV) [inline src/main/java/model/User.java]: If you create an inner class(enum) - don't save it among getters and setters.  Save it in a separate class or higher(before getters and setters) better

**Review by KrasuliaV — APPROVED** [review]

**Review by VasylMoshun — APPROVED** [review]

**Review by VasylMoshun — APPROVED** [review]

**Review by VasylMoshun — APPROVED** [review]

**Review by VasylMoshun — APPROVED** [review]

**Review by KrasuliaV — APPROVED** [review]

**Review by VasylMoshun — APPROVED** [review]

**Pull request merged**